scripts/PBAB+Collabs/zksync-testnet/contracts/Greeter.sol

```diff
@@ -0,0 +1,18 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
What's the context on this file?

Reply:
ah this was used for super early debug (lets see if deploying anything works) will remove 👍